addons/website/static/src/snippets/s_image_gallery/000.xml

```diff
@@ -14,7 +14,7 @@
             <div class="carousel-inner" style="padding: 0;">
                  <t t-foreach="images" t-as="image" t-key="image_index">
                     <div t-attf-class="carousel-item #{image_index == index and 'active' or None}">
-                        <img t-attf-class="#{attrClass || 'img img-fluid d-block'}" t-att-src="image.src" t-att-style="attrStyle" t-att-alt="image.alt" data-name="Image"/>
+                        <img class="img img-fluid d-block" t-att-src="image.src" t-att-alt="image.alt" data-name="Image" data-o-main-image="true"/>
```

Comment:
Do we need to render the images in qweb at all? If we end up replacing them with original images anyway, why not only rendering the carousel items and then add the original images in each of them?

Reply:
> we will probably have to modify the _onClickImg() of GalleryWidget to replace the images in the rendered template as well.

Indeed, we don't want that: if you set up an image wall snippet and add a border, rounded corners and other decorations on the snippet images, you don't want those for the slider when you click on those images.

But this can be improved anyway: make the qweb template more conditional:
- For the "option" case: you don't need the qweb to render images => don't do it
- For the slider outside of edit mode: you need the qweb to render images => let's keep it, for that case only